app/views/admin/exports/index.html.erb

```diff
@@ -27,5 +27,12 @@
     <%= link_to(users_admin_exports_path, class: "margin-bottom-2 button royal primary") do %>
         Download user data (CSV format)
     <% end %>
+
+    <h4>Signbank</h4>
+    <p>This export contains published sign data in CSV format, which can be used to import NZSL Share signs into Signbank. This export may take several seconds to complete.</p>
```

Comment:
Keep in mind that running in the background is also an option here, if it takes too long to complete in a request.